core/src/main/resources/hudson/model/View/sidepanel.jelly

```diff
@@ -43,14 +43,6 @@ THE SOFTWARE.
         <l:task href="${rootURL}/${it.viewUrl}newJob" icon="symbol-add" it="${app}" permission="${it.itemCreatePermission}" title="${%NewJob(it.newPronoun)}"/>
       </st:include>
 
-      <j:choose>
-        <j:when test="${it.class.name=='hudson.model.AllView'}">
-          <l:task href="${rootURL}/asynchPeople/" icon="icon-user icon-md" title="${%People}"/>
-        </j:when>
```

Comment:
FTR this is a bug IMO, as AllView can exist in folders. Note how https://weekly.ci.jenkins.io/job/folder/ links to `/asynchPeople/`.